apps/dapp-console-api/src/constants/faucetConfigs.ts

```diff
@@ -0,0 +1,192 @@
+import type { Address, Chain, Hex } from 'viem'
+import {
+  defineChain,
+  parseEther,
+  createPublicClient,
+  createWalletClient,
+  fallback,
+  http,
+} from 'viem'
+import {
+  baseSepolia,
+  modeTestnet,
+  optimismSepolia,
+  sepolia,
+  zoraSepolia,
+} from 'viem/chains'
+
+import { envVars } from '../constants'
+import { privateKeyToAccount } from 'viem/accounts'
+
+type FaucetConfig = {
+  // The chain id of the faucet (e.g. 420)
+  id: number
+  // Display name of the faucet in the UI (e.g. OP Goerli)
+  displayName: string
+  chain: Chain
+  // The amount of eth distributed for on-chain auth methods.
+  onChainDripAmount: bigint
+  // The amount of eth distributed for on-chain auth methods.
+  offChainDripAmount: bigint
+  // Environment that the faucet is supported in.
+  supportedEnvironments: Array<typeof envVars.DEPLOYMENT_ENV>
+  l1BridgeAddress?: Address
+  isL1Faucet?: boolean
+}
+
+// Configs for the Drippie contracts that help keep the faucet contract and admin wallets funded.
+type DrippieConfig = {
+  // Rpc urls for the chain the faucet is on.
+  rpcUrls: string[]
+  // Chain that the Drippie contract is deployed on.
+  chain: Chain
+  // Environment that the faucet is supported in.
+  supportedEnvironments: Array<typeof envVars.DEPLOYMENT_ENV>
+}
```

Comment:
nit: we should probably remove the DrippieConfig since it is not being used at the moment.